### PR TITLE
add NodeJS free-courses-pt_BR.md 

### PR DESCRIPTION
**Pull request by hpbonfim**
## What does this PR do?
Add resource(s)

### Description
add some NodeJS courses with a good pt BR content 

* [Serie API NodeJS](https://www.youtube.com/playlist?list=PL85ITvJ7FLoiXVwHXeOsOuVppGbBzo2dp) - Diego Fernandes (Rocketseat) (mini courses playlist with ~20m)
* [Testes no NodeJS aplicando TDD com Jest](https://www.youtube.com/watch?v=2G_mWfG0DZE&list=PL85ITvJ7FLoh7QBmTVzuNYvZaYPYwDmei) - Diego Fernandes (Rocketseat) (1h20 duration)

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -247,6 +247,9 @@
 * [Do Zero A Produção: Aprenda A Construir Uma API Node.Js Com Typescript](https://www.youtube.com/playlist?list=PLz_YTBuxtxt6_Zf1h-qzNsvVt46H8ziKh) - Waldemar Neto Dev Lab (YouTube)
 * [Imersão em desenvolvimento de APIs com Node.js](https://erickwendel.teachable.com/p/node-js-para-iniciantes-nodebr) - Erick Wendel (Teachable)
 * [RESTful com Node.js e Restify](https://www.youtube.com/playlist?list=PLy5T05I_eQYO5Y3S3kVqBxQzkUNllPazF) - Codecasts (YouTube)
+* [Serie API NodeJS](https://www.youtube.com/playlist?list=PL85ITvJ7FLoiXVwHXeOsOuVppGbBzo2dp) - Rocketseat (YouTube)
+* [Testes no NodeJS aplicando TDD com Jest | Diego Fernandes](https://www.youtube.com/watch?v=2G_mWfG0DZE&list=PL85ITvJ7FLoh7QBmTVzuNYvZaYPYwDmei) - Rocketseat (YouTube)
+* [TypeScript com NodeJS do jeito certo! | Diego Fernandes ](https://www.youtube.com/watch?v=aTf8QTjw4RE&list=PL85ITvJ7FLoh7QBmTVzuNYvZaYPYwDmei) - Rocketseat (YouTube)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:252]: not long enough for a non-playlist link

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -247,6 +247,9 @@
 * [Do Zero A Produção: Aprenda A Construir Uma API Node.Js Com Typescript](https://www.youtube.com/playlist?list=PLz_YTBuxtxt6_Zf1h-qzNsvVt46H8ziKh) - Waldemar Neto Dev Lab (YouTube)
 * [Imersão em desenvolvimento de APIs com Node.js](https://erickwendel.teachable.com/p/node-js-para-iniciantes-nodebr) - Erick Wendel (Teachable)
 * [RESTful com Node.js e Restify](https://www.youtube.com/playlist?list=PLy5T05I_eQYO5Y3S3kVqBxQzkUNllPazF) - Codecasts (YouTube)
+* [Serie API NodeJS](https://www.youtube.com/playlist?list=PL85ITvJ7FLoiXVwHXeOsOuVppGbBzo2dp) - Rocketseat (YouTube)
+* [Testes no NodeJS aplicando TDD com Jest | Diego Fernandes](https://www.youtube.com/watch?v=2G_mWfG0DZE&list=PL85ITvJ7FLoh7QBmTVzuNYvZaYPYwDmei) - Rocketseat (YouTube)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:251]:
```suggestion
* [Testes no NodeJS aplicando TDD com Jest](https://www.youtube.com/watch?v=2G_mWfG0DZE&list=PL85ITvJ7FLoh7QBmTVzuNYvZaYPYwDmei) - Diego Fernandes (Rocketseat)
```
please add in note in PR description if video is longer than an hour.
moving creator credit to creator field also avoids having to escape the pipe character

File: courses/free-courses-pt_BR.md [hunk 3]
```diff
@@ -247,6 +247,9 @@
 * [Do Zero A Produção: Aprenda A Construir Uma API Node.Js Com Typescript](https://www.youtube.com/playlist?list=PLz_YTBuxtxt6_Zf1h-qzNsvVt46H8ziKh) - Waldemar Neto Dev Lab (YouTube)
 * [Imersão em desenvolvimento de APIs com Node.js](https://erickwendel.teachable.com/p/node-js-para-iniciantes-nodebr) - Erick Wendel (Teachable)
 * [RESTful com Node.js e Restify](https://www.youtube.com/playlist?list=PLy5T05I_eQYO5Y3S3kVqBxQzkUNllPazF) - Codecasts (YouTube)
+* [Serie API NodeJS](https://www.youtube.com/playlist?list=PL85ITvJ7FLoiXVwHXeOsOuVppGbBzo2dp) - Rocketseat (YouTube)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:250]:
```suggestion
* [Serie API NodeJS](https://www.youtube.com/playlist?list=PL85ITvJ7FLoiXVwHXeOsOuVppGbBzo2dp) - Diego Fernandes (Rocketseat)
```

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-pt_BR.md (hunk 1 above)
Comment (henriquebonfim) [reply courses/free-courses-pt_BR.md:252]: ok, I'll remove the shortest video.

File: courses/free-courses-pt_BR.md (hunk 2 above)
Comment (henriquebonfim) [reply courses/free-courses-pt_BR.md:251]: ok. done!

**Comment by hpbonfim** [comment]
Details:

* [Serie API NodeJS](https://www.youtube.com/playlist?list=PL85ITvJ7FLoiXVwHXeOsOuVppGbBzo2dp) - Diego Fernandes (Rocketseat) (has many short courses with ~20m)
* [Testes no NodeJS aplicando TDD com Jest](https://www.youtube.com/watch?v=2G_mWfG0DZE&list=PL85ITvJ7FLoh7QBmTVzuNYvZaYPYwDmei) - Diego Fernandes (Rocketseat) (1h20 duration)

**Pull request merged**